src/utils.js

```diff
@@ -70,7 +70,31 @@ const isTriggerKey = ({ triggerKeys, action, key }) => (
   triggerKeys[action].indexOf(key) > -1
 );
 
+// NOTE: These are the breakpoints used in Bootstrap v4.0.0 as seen in
+// the documentation (https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints)
+const breakpoints = {
```

Comment:
Including this here so we can use Bootstrap breakpoints in React code, e.g. with `react-responsive`.